### ISSUE TITLE
New version of tiny-slider on npm. ngx-tiny-slider dont work any more correctly

### ISSUE THREAD

**Issue by SvenBudak** [vdanyliv/ngx-tiny-slider#30]
Hi! :)

There is a new version from tiny-slider on npm. i get some new errors now:

```
ERROR in node_modules/tiny-slider/src/tiny-slider.d.ts:243:5 - error TS2300: Duplicate identifier 'preventScrollOnTouch'.

243     preventScrollOnTouch?: "auto" | "force" | false;
        ~~~~~~~~~~~~~~~~~~~~
node_modules/tiny-slider/src/tiny-slider.d.ts:260:5 - error TS2300: Duplicate identifier 'preventScrollOnTouch'.

260     preventScrollOnTouch?: "auto" | "force" | false;
```

maybe there are more problems, but this is what i get.
you can see the issue post about the update here: https://github.com/ganlanyuan/tiny-slider/issues/588

best regards

**Comment by Duarte10** [comment]
Having the same issue, if anyone found a solution please share it :)

**Comment by Duarte10** [comment]
> Having the same issue, if anyone found a solution please share it :)

Fixed it in my angular project by going to the package.json file and specifying the tiny-slider version prior to the latest release.

**Comment by SvenBudak** [comment]
@Duarte10 this is not a fix... we are all happy that tiny-slider gets after 2 years a update to npm lib.

**Comment by Duarte10** [comment]
> @Duarte10 this is not a fix... we are all happy that tiny-slider gets after 2 years a update to npm lib.

That's why I said I fixed it in my angular project, if someone else also has an app in production that uses ngx-tiny-slider and doesn't have the time to wait for a new ngx-tiny-slider release that accommodates the changes in the recent tiny-slider release to npm, they can apply this "fix" to their project.

**Comment by vdanyliv** [comment]
the latest release of tiny-slider didn't fix the problem

**Comment by SvenBudak** [comment]
Thats sad.. but you anyway plan to make a update for `ngx-tiny-slider`?

**Comment by vdanyliv** [comment]
Of-course @SvenBudak

**Comment by vdanyliv** [comment]
new PR raised https://github.com/ganlanyuan/tiny-slider/pull/598

**Comment by ttong9** [comment]
Hi, is there any update on this bugfix?



> That's why I said I fixed it in my angular project, if someone else also has an app in production that uses ngx-tiny-slider and doesn't have the time to wait for a new ngx-tiny-slider release that accommodates the changes in the recent tiny-slider release to npm, they can apply this "fix" to their project.

In the meantime, @Duarte10 can you please confirm which version worked for you?

**Comment by Duarte10** [comment]
> 
> 
> Hi, is there any update on this bugfix?
> 
> > That's why I said I fixed it in my angular project, if someone else also has an app in production that uses ngx-tiny-slider and doesn't have the time to wait for a new ngx-tiny-slider release that accommodates the changes in the recent tiny-slider release to npm, they can apply this "fix" to their project.
> 
> In the meantime, @Duarte10 can you please confirm which version worked for you?

2.9.2

**Comment by vdanyliv** [comment]
> Hi, is there any update on this bugfix?
> 
> > That's why I said I fixed it in my angular project, if someone else also has an app in production that uses ngx-tiny-slider and doesn't have the time to wait for a new ngx-tiny-slider release that accommodates the changes in the recent tiny-slider release to npm, they can apply this "fix" to their project.
> 
> In the meantime, @Duarte10 can you please confirm which version worked for you?

no updates

**Comment by ttong9** [comment]
thank you

**Comment by vdanyliv** [comment]
@SvenBudak @ttong9 @Duarte10 what if I create a fork of the original tiny-slider and we start fixing bugs more often?
What do you think guys?

**Comment by SvenBudak** [comment]
I don't think that I am a big help. My logic skills are limited. I am more specialized in UX and UI. But I would still be happy about an updated version :)

**Comment by Paul75** [comment]
same problem and not fixed ....
